docker-app/qfieldcloud/core/admin.py

```diff
@@ -71,6 +95,104 @@ def admin_urlname_by_obj(value, arg):
 admin.site.unregister(SocialAccount)
 admin.site.unregister(SocialApp)
 admin.site.unregister(SocialToken)
+admin.site.unregister(EmailAddress)
+
+
+class EmailAddressAdmin(allauth_account_admin.EmailAddressAdmin):
```

Comment:
Can we move this more to the bottom of the whole `admin.py` module. It's quite far from being the most imporant thing to begin with it.